src/azure/azureTools.ts

```diff
@@ -378,6 +378,22 @@ class AzureTools {
 		}
 	}
 
+
+	async deleteKustomization(
+		kustomizationName: string,
+		contextName: string,
+		clusterProvider: AzureClusterProvider,
+	) {
+		const deleteSourceShellResult = await this.invokeAzCommand(
+			`az k8s-configuration flux kustomization delete -n ${kustomizationName} --yes`,
+			contextName,
+			clusterProvider,
+		);
+		if (deleteSourceShellResult?.code !== 0) {
+			telemetry.sendError(TelemetryErrorEventNames.FAILED_TO_RUN_AZ_DELETE_SOURCE);
```

Comment:
Should this be a "delete az workload" telemetry event & variable name?

Reply:
You're right, this needs changes. And some new constants to be declared as well 👍